### PR TITLE
Reworked Example on :class:~.Polygon

### PR DESCRIPTION
**Pull request by peaceheis**
## Changelog / Overview

Added a more detailed example for the Polygon Class

## Motivation

The previous example did not have as extensive showcasing of Polygon features.
## Explanation for Changes

[Documentation Reference](https://manimce--####.org.readthedocs.build)

## Checklist
- [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [ ] I have written a descriptive PR title (see top of PR template for examples)
- [ ] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: manim/mobject/geometry.py [hunk 1]
```diff
@@ -1713,19 +1713,26 @@ class Polygon(Polygram):
...
+            pentagon.set_color([TEAL, RED, YELLOW])
+            pentagon.set_stroke(width=8)
+            rectangle = Polygon([-2.75, -1.75, 0], [-2.75, -0.75, 0], [2.50, -0.75, 0], [3, -1.75, 0])
+            rectangle.set_color(GREEN)
+            rectangle.set_fill(WHITE, opacity=1)
+            rectangle.set_stroke(width=10)
+            upright_rectangle = Polygon([2.15, -2, 0], [3.5, -2, 0], [3.5, 2.5, 0], [2.15, 2.5, 0])
+            upright_rectangle.set_color(BLACK)
+            upright_rectangle.set_fill(PURPLE, opacity=0.5)
+            self.add(isosceles, square, pentagon, rectangle, upright_rectangle)
```
Comment (Nikhil0504) [inline manim/mobject/geometry.py:1735]: Check the indentation again.

**Review by Nikhil0504 — CHANGES_REQUESTED** [review]

**Review by naveen521kk — CHANGES_REQUESTED** [review]
Can you follow the instructions in [#hackthon-guidelines](https://discord.com/channels/581738731934056449/846805485529464832/849962667568529428)? ie, use the manimator bot(`!mdocstring` command) to create doc strings.

**Comment by peaceheis** [comment]
> Can you follow the instructions in [#hackthon-guidelines](https://discord.com/channels/581738731934056449/846805485529464832/849962667568529428)? ie, use the manimator bot(`!mdocstring` command) to create doc strings.

I did use the doc string command, Im not sure what went wrong..

**Comment by naveen521kk** [comment]
> I did use the doc string command, Im not sure what went wrong..

You should paste that as it is.

**Comment by Nikhil0504** [comment]
> > Can you follow the instructions in [#hackthon-guidelines](https://discord.com/channels/581738731934056449/846805485529464832/849962667568529428)? ie, use the manimator bot(`!mdocstring` command) to create doc strings.
> 
> I did use the doc string command, Im not sure what went wrong..

You copy pasted it but only the first line is indented correctly, you should tab in the other lines as copy paste messes them up.

**Comment by hydrobeam** [comment]
Tbh I prefer the old example more than this one. It feels less visually cluttered. I do realize that `Polygon` was under `Needs rework` for the hackathon, but I'm not sure in which direction it was intended for the example to go.

**Comment by peaceheis** [comment]
> Tbh I prefer the old example more than this one. It feels less visually cluttered. I do realize that `Polygon` was under `Needs rework` for the hackathon, but I'm not sure in which direction it was intended for the example to go.

That's a fair assessment, I did err on the side of too much representation and not too little.

**Comment by hydrobeam** [comment]
Closing this since there hasn't been a lot of activity and the current example is more clear as to what `Polygon` should do. Thank you for the effort nonetheless!

**Pull request closed without merging**